src/select/Select.ts

```diff
@@ -288,15 +288,15 @@ export default class Select extends SelectBase<SelectProperties> {
 				onfocus: this._onFocus,
 				onkeydown: this._onKeyDown
 			}, optionNodes),
-			v('span', { classes: this.classes(css.nativeArrow) }, [
+			multiple ? null : v('span', { classes: this.classes(css.arrow) }, [
```

Comment:
It's a `<select>` element so it's labeled by its `<label>`. The arrow here is just extra visual window dressing :)